src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -129,6 +129,11 @@ internal static string ReturnCommonPathRootFromTargetSpecifiersIfOneExists(IEnum
 
         public override int Run(AnalyzeOptions analyzeOptions)
         {
+            if (analyzeOptions.TargetFileSpecifiers == null || !analyzeOptions.TargetFileSpecifiers.Any())
+            {
+                throw new ArgumentNullException("Please specify one or more files, directories, or filter patterns for BinSkim analyze.", (Exception)null);
```

Comment:
>null [](http://example.com/codeflow?start=149&length=4)

You don't need to pass this argument, null is the default value for inner exception when it isn't provided.

Reply:
If only passing one argument, it will be treated as the parameter name.
![image](https://user-images.githubusercontent.com/119978670/207407712-9306472a-d9c7-435e-9322-38f6b845cec5.png)
Then the result will be as follows:
![image](https://user-images.githubusercontent.com/119978670/207407801-a40628be-1c62-420f-aecb-a641ea95e791.png)

Reply:
Oh, right, this is one of those exception ctors that's inconsistent in .NET.

So do this:

throw new ArgumentNullException(nameof(analyzeOptions.TargetFileSpecifiers));

Reply:
Ok. 
Now the result is as follows:
<img width="1175" alt="image" src="https://user-images.githubusercontent.com/119978670/207425731-255bb9e6-8245-4203-8fb9-745ad51c967f.png">